docs/api.md

```diff
@@ -21,10 +21,12 @@ The auro-menu element provides users a way to select from a list of options.
 
 ## Events
 
-| Event                        | Type               | Description                                      |
-|------------------------------|--------------------|--------------------------------------------------|
-| `auroMenuSelectValueFailure` | `CustomEvent<any>` | Notifies that a an attempt to select a menuoption by matching a value has failed. |
-| `selectedOption`             | `CustomEvent<any>` | Notifies that a new menuoption selection has been made. |
+| Event                        | Type                   | Description                                      |
+|------------------------------|------------------------|--------------------------------------------------|
+| `auroMenuActivatedOption`    | `CustomEvent<Element>` | Notifies that a menuoption has been made `active`. |
```

Comment:
I feel like we had a conversation about this? I am partial to the idea of prefixing custom events with `auro[name]` to make it clear that this is a synthetic event specific to an auro element. 

Wondering if there are other elements we should update this way?

Reply:
I would be in favor of that being a standard. I am hesitant to make that change right this moment for the pre-existing events because it means taking on a change for auro-select as well to handle the breaking change. 

Perhaps, another release to update all three components at once for that?